### PR TITLE
Control the Kernel

### PR DESCRIPTION
**Pull request by johnisanerd**
This PR makes sure that folks using the GrovePi and installing it themselves control the Kernel version until a fix is found.

### CODE REVIEW

**Comment by johnisanerd** [comment]
I think we are waiting to hear about Roberts investigation.  Yesterday I
tried with stretch and the previously non working commands seem to be
working with the GrovePi.  Still fails for the line follower though.

**Comment by johnisanerd** [comment]
@RobertLucian is this dead and done now, no longer needed, right?

File: Script/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+GrovePi
```
Comment (johnisanerd) [inline Script/README.md]: @RobertLucian I think we should keep this, but maybe delete or reject the changes below right?

Comment (RobertLucian) [reply Script/README.md]: I would just close the PR and in case we need it, we can just open it up.

**Pull request closed without merging**